model/src/main/kotlin/config/ScanStorageConfiguration.kt

```diff
@@ -67,14 +67,24 @@ data class FileBasedStorageConfiguration(
  */
 data class PostgresStorageConfiguration(
     /**
-     * The database URL in JDBC format.
+     * Hostname or IP of postgres server
```

Comment:
In general, please write all docs like you'd write sentences in a printed book. I.e. use a dot at the end of a sentence.